src/solver.cpp

```diff
@@ -635,18 +635,6 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
         registry, ++id, Primitive::Normalization, layernorm::Layernorm4DCKForward{}.SolverDbId());
     Register(registry, ++id, Primitive::Normalization, layernorm::LayernormForward{}.SolverDbId());
     Register(registry, ++id, Primitive::Reduce, reduce::SumForward{}.SolverDbId());
-    RegisterWithSolver(registry,
-                       ++id,
-                       conv::ConvHipImplicitGemmF16F8F16FwdXdlops{},
-                       miopenConvolutionAlgoImplicitGEMM);
-    RegisterWithSolver(registry,
-                       ++id,
-                       conv::ConvHipImplicitGemmF16F8F16BwdXdlops{},
-                       miopenConvolutionAlgoImplicitGEMM);
-    RegisterWithSolver(registry,
-                       ++id,
-                       conv::ConvHipImplicitGemmF16F8F16WrwXdlops{},
-                       miopenConvolutionAlgoImplicitGEMM);
```

Comment:
What impact does the solver ID have on our caches?
Might need to double check if this will cause problems since it'll shift the ID for all the solvers below.

Reply:
Yea I thought I remembered seeing this:

  // When solver gets removed its registration line should be replaced with ++id to keep
    // backwards compatibility. New solvers should only be added to the end of list unless it is
    // intended to reuse an id of a removed solver.
    
If we remove these in this PR, then I think we need to keep the ID's still incrementing with a note.

Reply:
@BrianHarrisonAMD  oh, yes. The ++id needs to be placed. I will fix it.